content/en/docs/setup/production-environment/turnkey-solutions.md

```diff
@@ -7,7 +7,8 @@ weight: 30
 
 This page provides a list of Kubernetes certified solution providers. From each
 provider page, you can learn how to install and setup production
-ready clusters.
+ready clusters. Please go through [CNCF Cloud Native Interactive Landscape](https://landscape.cncf.io/) to have a better understanding.
```

Comment:
It is desirable to wrap long lines like this properly.

Reply:
@tengqm Like how? What changes do you expect?

Reply:
I'm not sure how useful by adding this sentence.

```suggestion
ready clusters. Please go through the [CNCF Cloud Native Landscape](https://landscape.cncf.io/)
to have a better understanding.
```